Library/ViewModels/ProjectPamphletContentViewModel.swift

```diff
@@ -104,11 +109,48 @@ public final class ProjectPamphletContentViewModel: ProjectPamphletContentViewMo
       projectCreatorDetailsExperimentValues
     )
 
+    let projectSummaryRequestValues = project
+      .take(first: 1)
+      .map(\.slug)
+      .switchMap { slug in
+        AppEnvironment.current.apiService
+          .fetchProjectSummary(query: projectSummaryQuery(withSlug: slug))
+          .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
+          .map { $0.projectSummary }
+          .prefix(value: [])
+          .demoteErrors(replaceErrorWith: [])
+          .materialize()
+      }
+      .values()
+
+    let projectSummaryHasLoadedValues = projectSummaryRequestValues.filter { $0.isEmpty == false }
+
+    let projectSummaryExperimentValues = projectAndRefTag
+      .takePairWhen(projectSummaryHasLoadedValues)
+      .map { projectAndRefTag, projectSummaryItems in
+        (projectAndRefTag.0, projectAndRefTag.1, projectSummaryItems)
+      }
+      .map { project, refTag, projectSummaryItems -> [ProjectSummaryEnvelope.ProjectSummaryItem] in
+        guard
+          projectPageConversionProjectSummaryExperiment(project: project, refTag: refTag) != .control
```

Comment:
Hmm I'm wondering if we can move this optimizely experiment variant check higher up in the logic chain. It seems a bit wasteful to make the network request for the project summary items before we know whether we'll even display them. 🤔

Reply:
Yeah we did this with this experiment and the creator details one. I guess the only risk is that we could activate the experiment without knowing if the request for the data succeeded or not. So here we're flipping that by first ensuring that we have the data to display before activating 🤷‍♂

Reply:
You _could_ call `getVariation` which wouldn't activate the experiment. 🤔